combat-trainer.lic

```diff
@@ -138,7 +138,19 @@ class SetupProcess
       echo('skipping summoned weapons because no moonblade available') if $debug_mode_ct
       weapon_training = weapon_training.reject { |skill, _| game_state.summoned_info(skill) }
     end
-    new_weapon_skill = weapon_training.min_by { |skill, _| [DRSkill.getxp(skill), DRSkill.getrank(skill)] }.first
+
+    new_weapon_skill=""
+    echo "Priority weapons: #{get_settings.priority_weapons}" if $debug_mode_ct
```

Comment:
This should be in `initialize`

Reply:
I wouldn't think so.  new_weapon_skill should be scoped locally to the method, not to the class

Reply:
He means the echo, you're echoing out the fact that priority weapons is set every time that determine next to train is called, instead of once when the setup class is initialized.